posthog/warehouse/api/view_link.py

```diff
@@ -90,4 +90,14 @@ class ViewLinkViewSet(TeamAndOrgViewSetMixin, viewsets.ModelViewSet):
     ordering = "-created_at"
 
     def safely_get_queryset(self, queryset):
-        return queryset.exclude(deleted=True).prefetch_related("created_by").order_by(self.ordering)
+        return queryset.prefetch_related("created_by").order_by(self.ordering)
```

Comment:
Why do we not want the `.exclude(deleted=True)` on the queryset? That way all usages don't need to implement it themselves, right?